app/assets/stylesheets/styles/components/_nav.css.scss

```diff
@@ -66,12 +72,12 @@ html, body {
         }
 
         &:hover {
-          background-color: rgba(0,0,0,0.5) !important;
+          background-color: rgba(0,0,0,0.8) !important;
```

Comment:
!important should not be used<br>`0.8` should be written without a leading zero as `.8`<br>Commas in function arguments should be followed by a single space